content/en/docs/concepts/windows/intro.md

```diff
@@ -352,6 +352,37 @@ Windows Server SAC release
 
 The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
 
+## Hardware recommendations and considerations {#windows-hardware-recommendations}
+
+{{< note >}}
+The following hardware requirements are _recommended_ values and do not represent
+minimum requirements. Refer to [Production worker nodes Kubernetes documentation](https://kubernetes.io/docs/setup/production-environment/#production-worker-nodes)
+for guidance on deciding on resources for production worker nodes.
+{{< /note >}}
+
+- 64-bit processor 4 CPU cores or more, capable of supporting virtualization
+- 8GB or more of RAM
+- 50GB or more of free disk space
+
+Refer to the specific Windows Server OS documentation for the most
+up-to-date information on minimum hardware requirements. Refer to
+[Hardware requirements for Windows Server Microsoft documentation](https://learn.microsoft.com/en-us/windows-server/get-started/hardware-requirements)
+for the minimum hardware requirements. 
+
+To optimize system resources, if a graphical user interface is not required,
+it may be preferable to use a Windows Server OS installation that excludes
+the [Windows Desktop Experience](https://learn.microsoft.com/en-us/windows-server/get-started/install-options-server-core-desktop-experience)
+installation option, as this configuration typically frees up more system 
+resources. 
+
+When it comes to considering disk space, compared to Linux worker nodes, Windows container images typically exceed the
+sizes of Linux images, with sizes ranging from
+[300MB to as much 11GB](https://techcommunity.microsoft.com/t5/containers/nano-server-x-server-core-x-server-which-base-image-is-the-right/ba-p/2835785)
+for a single image. Additionally, take note that the `C:` drive in a Windows container represents a virtual free size of
+20GB by default. See
```

Comment:
We should clarify that this does not consume 20gigs when a container starts, it is the size of which the disk for a single container can grow up too, if container local storage is used.